### PR TITLE
Added cimode to supported languages

### PR DESCRIPTION
**Pull request by nikomakela**
PT-1016 Added cimode to supported languages, so it could be used for debugging and as a helper tool when a translation office or some translator works with translations.

image

### CODE REVIEW

**Review by melniiv — APPROVED** [review]

**Comment by nikomakela** [comment]
Added a feature flag to show CIMODE in a language selector: 
image

The feature is **off in production** server and **on in staging** server.

**Comment by nikomakela** [comment]
Same feature in Palvelutarjotin admin https://github.com/City-of-Helsinki/palvelutarjotin-admin/pull/177

**Comment by nikomakela** [comment]
It seems like the feature flag did not work in review but not in staging either!

File: src/domain/app/header/__tests__/Header.test.tsx [hunk 1]
```diff
@@ -14,3 +14,28 @@ it('Header matches snapshot', () => {
...
+  });
+
+  afterAll(() => {
+    process.env = OLD_ENV; // Restore old environment
+  });
+
+  test('cimode is rendered in a list of languages when feature flag is on', async () => {
+    process.env.NEXT_PUBLIC_LANGUAGE_CIMODE_VISIBLE = 'true';
+    const { container } = render(<Header />);
+    await findAllByText(container, /CIMODE/);
```
Comment (Sartonon) [inline src/domain/app/header/__tests__/Header.test.tsx:33]: Why isn't screen.findAllByText used?

File: src/domain/app/header/__tests__/Header.test.tsx [hunk 2]
```diff
@@ -14,3 +14,28 @@ it('Header matches snapshot', () => {
...
+  });
+
+  test('cimode is not rendered in a list of languages when feature flag is off', async () => {
+    process.env.NEXT_PUBLIC_LANGUAGE_CIMODE_VISIBLE = 'false';
+    const { container } = render(<Header />);
+    expect(queryAllByText(container, /CIMODE/).length).toBe(0);
```
Comment (Sartonon) [inline src/domain/app/header/__tests__/Header.test.tsx:39]: Same here, screen could be used

File: .env.local.example [hunk 3]
```diff
@@ -3,4 +3,5 @@ NEXT_PUBLIC_CAPTCHA_KEY=
 
 NEXT_PUBLIC_MATOMO_URL_BASE=https://analytics.hel.ninja/
 NEXT_PUBLIC_MATOMO_SITE_ID=68
-NEXT_PUBLIC_MATOMO_ENABLED=false
\ No newline at end of file
+NEXT_PUBLIC_MATOMO_ENABLED=false
+NEXT_PUBLIC_LANGUAGE_CIMODE_VISIBLE=false
```
Comment (Sartonon) [inline .env.local.example:7]: Should be added to Dockerfile ARG also

Comment (nikomakela) [reply .env.local.example:7]: https://github.com/City-of-Helsinki/palvelutarjotin-ui/pull/120

**Comment by nikomakela** [comment]
I will fix these here https://github.com/City-of-Helsinki/palvelutarjotin-ui/pull/120

**Pull request merged**